torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19497,6 +19504,15 @@ def __init__(
         "_refs.nn.functional.elu",
         torch_opinfo_name="nn.functional.elu",
     ),
+    PythonRefInfo(
+        "_refs.nn.functional.gelu",
+        torch_opinfo_name="nn.functional.gelu",
+        decorators=(
+            # Need FakeTensor support for meta coverage
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_meta',),
+            DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref',),
```

Comment:
Wait why is this test failing?

Reply:
Currently we don't rebind `torch.Tensor.mul` to `_refs.mul`. It should pass after that.

I originally wrote this with `torch.mul` to make it pass, but it's actually just too ugly to handle.

Reply:
Actually, don't see any reason why it isn't done today. I added it to the PR. (cc: @ezyang )